src/tasks/EmitBundleTask/Templates/mono-bundled-satellite-assembly.c

```diff
@@ -0,0 +1,6 @@
+const MonoBundledSatelliteAssemblyResource %Symbol% = {
+    .resource = { .type = MONO_BUNDLED_SATELLITE_ASSEMBLY },
+    .satellite_assembly = { .name = "%RegisteredName%",
+                            .data = %Symbol%_data,
+                            .size = %Len% },
```

Comment:
Don't you also need to initialize `.culture` ?

Reply:
If we agree that satellites should have a name that is `culture + / + name` and drop its culture field of the struct since it is not really used when bundling, then the name should be pre-allocated to be `culture + / + name`.

Reply:
Good catch, added in `.culture` to this template that will be populated through the EmitBundle task